.github/workflows/ci-java.yaml

```diff
@@ -19,7 +17,7 @@ jobs:
       - name: Setup Java
         uses: actions/setup-java@v1
         with:
-          java-version: '11.0.12' # The JDK version to make available on the path.
+          java-version: '17' # The JDK version to make available on the path.
```

Comment:
Both `ci-java` and `ci-java8` seems the same except compile target profile. I can prepare a PR to reuse both workflows into a single one in you want.